dojo/tools/veracode_sca/parser.py

```diff
@@ -51,7 +52,15 @@ def _get_findings_json(self, file, test):
             if issue.get("issue_type") != "vulnerability":
                 continue
 
-            date = parser.parse(issue.get("created_date"))
+            # Get the date based on the first_seen setting
+            try:
+                if settings.USE_FIRST_SEEN:
+                    date = parser.parse(issue.get("created_date"))
+                else:
+                    date = parser.parse(issue.get("created_date"))
```

Comment:
These looks same for me

Reply:
They are the same, but this time it is intentional. In the unit test reports for this parser, there is not any available data in the report to satisfy the notion of a "last seen" date. Before this change, the `created_date` field was used, so it seemed appropriate to maintain that same behavior for both cases.

I am open to any suggestions that could make more sense though!

Reply:
In that case, is it necessary to replace the original implementation if there is no change in behavior?